bin/build

```diff
@@ -10,6 +10,8 @@ printf '{"commit":"%s","version":"%s","source":"https://github.com/%s/%s","build
   "$CIRCLE_BUILD_URL" \
 > version.json
 
+cp .env-dist .env
```

Comment:
@whd just to confirm, none of the variables in .env-dist would be applicable for a production deployment?